phase1/openstack/Kconfig

```diff
@@ -0,0 +1,59 @@
+menu "Openstack configuration"
+
+config phase1.openstack.os_image
+	string "OS image for nodes"
+	default "ubuntu-16.04"
+	help
+	  The OS image to use for deploying vm.
+
+config phase1.openstack.flavor_name
+	string "flavor name"
+	default "m1.medium"
+	help
+	  The flavor to use for the vm instance.
+
+config phase1.openstack.ssh_key_file
+	string "path to ssh private key file"
+	default "~/.ssh/id_rsa.terraform"
+	help
+	  The path to ssh private key file with which to configure
+	  the vm for auto-login. The public key file should also
+	  exist at the same path prefixed by .pub.
+
+config phase1.openstack.project
+	string "Openstack tenant"
+	help
+	  The Openstack tenant/project in which to deploy vm's.
+
+config phase1.openstack.region
+	string "region"
+	default "us-central1"
```

Comment:
Remove the default region? as that may not be present in all openstack installations?

Reply:
okie, will remove and test.